api/utilities.py

```diff
@@ -1,10 +1,34 @@
-
+from requests import get, post
 import facebook as fb
 from django.http import HttpResponse
-
 import os
 from dotenv import load_dotenv
 
+def create_discord_event(name, description, start_time, end_time, location, image):
+    acm_guild_id = ""
```

Comment:
Even if you're only going to load in a .env file later when you're testing, I think you should change acm_guild_id and bot_token to load it in from the .env now so that you don't have to change it later. That way as well, we won't need to change code once we add in a `.env` file locally.

Reply:
Done!